addons/test_mail/tests/test_mail_followers_unsubscribe.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.test_mail.tests.test_message_post import TestMessagePostCommon
+from odoo.tests import tagged
+from odoo.tests import users
+
+
+@tagged('mail_followers', 'post_install', '-at_install')
+class UnfollowFromInboxTest(TestMessagePostCommon):
```

Comment:
Yes, indeed, it’s more logical as there are already a lot of tests there. So, I have moved the tests there. Also, moved the test in a separate commit because it is hard to maintain them in different commit.